app/templates/base.html

```diff
@@ -18,7 +18,7 @@
 
 </head>
 <body>
-    {% if session['logged_in'] %}
+    {% if session.get('logged_in') %}
```

Comment:
When accessing keys of a dict we should always use `.get()` as it wont throw a `KeyError` if it isn't there